lib/openstack/openstack_handle/handle.rb

```diff
@@ -25,16 +25,19 @@ class Handle
       "Orchestration" => :orchestration
     }
 
+    def self.try_connection_with_ssl
```

Comment:
So, I understand the problem and we can't fix the lack of a SSL/verify ssl mode option for the endpoint right now but I think it might make more sense to just call this method `try_connection` and have the existing logic from this PR stay the same.

I'd rather we spend the time on removing this complexity by adding a SSL/ ssl verify mode option to the endpoint in the UI/backend.